src/test/progpow_tests.cpp

```diff
@@ -0,0 +1,166 @@
+#include <test/test_veil.h>
```

Comment:
At the top of the file add 
// Copyright (c) 2019 Veil developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.